EngineLayer/SpectralLibrarySearch/SpectralLibrary.cs

```diff
@@ -360,7 +495,15 @@ private void IndexSpectralLibrary(string path)
                     reader.DiscardBufferedData();
 
                     // parse the header
-                    var libraryItem = ReadLibrarySpectrum(reader, onlyReadHeader: true);
+                    LibrarySpectrum libraryItem;
+                    if (path.Contains("pdeep"))
```

Comment:
this seems a bit fragile. i can imagine users using a pdeep library but not having "pdeep" in the filename, and then getting a crash. but it would require a larger code rewrite to do it my preferred way (looking for "Spec=pDeep" in the comment field).